x-pack/plugins/index_management/public/application/sections/home/enrich_policies_list/policies_table/policies_table.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { EuiInMemoryTable, EuiBasicTableColumn, EuiSearchBarProps, EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { SerializedEnrichPolicy } from '../../../../../../common/types';
+
+export interface Props {
+  policies: SerializedEnrichPolicy[];
+  onReloadClick: () => void;
+  onDeletePolicyClick: (policyName: string) => void;
+  onExecutePolicyClick: (policyName: string) => void;
+}
+
+const pagination = {
+  initialPageSize: 5,
+  pageSizeOptions: [3, 5, 8],
```

Comment:
these values were probably used for testing, should we set them to something larger?

Reply:
Nice one! 

I'll replace it with the following, which seemed to have been the agreement from the `default rows per page setting` issue
```
const pagination = {
  initialPageSize: 50,
  pageSizeOptions: [25, 50, 100],
};
```